### PR TITLE
Adding missing sections for `container*`

### LINKED ISSUE #24786: syntax is missing
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/container-type

### What specific section or headline is this issue about?

missing formal definition and syntax

### What information was incorrect, unhelpful, or incomplete?

two sections of the doc are missing

### What did you expect to see?

formal definion and syntax

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/container-type`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/container-type
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/container-type/index.md
* Last commit: https://github.com/mdn/content/commit/489dbd4d8e2e099733d3bb04ad3f97678c54ac8e
* Document last modified: 2023-02-21T22:18:29.000Z

</details>

### PR DESCRIPTION
**Pull request by yarusome**
### Description

This PR corrects the "Syntax" section and adds the "Formal definition" and "Formal syntax" sections for the `container`, `container-name`, and `container-type` properties.

### Additional details

The syntax examples are adapted from the [spec](https://w3c.github.io/csswg-drafts/css-contain-3/#container-queries).

### Related issues and pull requests

Fixes #24786.
**Depends on:** https://github.com/mdn/data/pull/657.

### CODE REVIEW

**Comment by yarusome** [comment]
@estelle They'll be there once https://github.com/mdn/data/pull/657 has landed *and* mdn/data has a new release.

File: files/en-us/web/css/container-name/index.md [hunk 1]
```diff
@@ -13,8 +13,19 @@ When a containment context is given a name, it can be specifically targeted usin
...
 ## Syntax
 
-```plain
-container-name: <container-name>;
+```css
+/* A single name */
+container-name: my-layout;
+
+/* Multiple names */
+container-name: my-page-layout my-component-library;
```
Comment (estelle) [inline files/en-us/web/css/container-name/index.md:21]:
```suggestion
/* A single name */
container-name: myLayout;

/* Multiple names */
container-name: myPageLayout myComponentLibrary;
```

Comment (estelle) [reply files/en-us/web/css/container-name/index.md:21]: changing the names so i can create a commit and refresh the previews.

**Comment by estelle** [comment]
The formal definitions are still not showing up. Otherwise, approved.

**Review by estelle — APPROVED** [review]
The formal definitions are missing.

**Comment by yarusome** [comment]
I don't know why it's still missing after a new [release](https://github.com/mdn/data/releases/tag/v2.0.32) containing relevant data has already published :/

**Comment by teoli2003** [comment]
I'm not sure a new mdn/data has been actually published. @schalkneethling Can you have a quick look?

**Comment by estelle** [comment]
@schalkneethling Can you take a look and confirm mdn/data has been published?

**Comment by bsmth** [comment]
Thanks to @caugner for looking into it. This PR __depends on__ https://github.com/mdn/yari/pull/8759

It requires some work to migrate the CSSSyntax macro to use v6 of `@webref/css`.

**Comment by yarusome** [comment]
@bsmth Just curious: Why does `{{CSSInfo}}` also needs migration? It doesn't seem to have `webref/css` as a dependency. (I *may* work on a fix after the ongoing color-related issues if I can figure out what these two macros do internally.)

**Comment by caugner** [comment]
Oh, so we really "just" need an `mdn-data` release?

**Comment by caugner** [comment]
@bsmth `mdn-data` [2.0.32](https://www.npmjs.com/package/mdn-data) is now published! 🎉

**Comment by bsmth** [comment]
Thanks for investigating and fixing, Claas

**Pull request merged**